anybox/recipe/odoo/server.py

```diff
@@ -54,6 +54,28 @@ def __init__(self, *a, **kw):
             sw_modules = ('web', ) + sw_modules
         self.server_wide_modules = sw_modules
 
+        def replace_python_header(new_header):
+            """ Update the script template with the new header. """
+            zc.buildout.easy_install.script_template = (
+                zc.buildout.easy_install.script_template.replace(
+                    zc.buildout.easy_install.script_header, new_header))
+            zc.buildout.easy_install.py_script_template = (
+                zc.buildout.easy_install.py_script_template.replace(
+                    zc.buildout.easy_install.script_header, new_header))
+
+        if self.python_scripts_executable:
+            # Monkeypatch the script headers to replace the python
+            # executable by the one configured by the user
+            new_header = '#!%s' % self.python_scripts_executable
+            replace_python_header(new_header)
+
+        elif self.buildout_dir in sys.executable:
```

Comment:
Nice! In our particular use case, the virtualenv is not available on the production systems, and we set `python-scripts-executable = /usr/bin/env python`, but I can see this work in other cases. Maybe you should only execute this if the relative-paths option is activated, and in the lines below, let os.path.relpath do the substitution of the executable path?

Reply:
Agreed! I'll update it